src/server.js

```diff
@@ -70,11 +71,16 @@ app.use((error, request, response) => {
 	});
 });
 
-// Starts the server
-logger.info('Starting server');
+async function main() {
+	// Starts the server
+	logger.info('Starting server');
+
+	await database.connect();
+	await cache.connect();
 
-database.connect().then(() => {
 	app.listen(port, () => {
 		logger.success(`Server started on port ${port}`);
 	});
-});
\ No newline at end of file
+}
+
+main();
```

Comment:
I recommend doing a `.catch(err=>console.error(err))` here so you nodejs don't yell at you in newer versions.
in some versions I doesnt even log the error if you dont catch

Reply:
Fixed